examples/basics/plotting/grid_x_y_viewbox.py

```diff
@@ -0,0 +1,48 @@
+
+
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""
++---+-------------+
+|   |             |
+| y |   viewbox   |
+|   |             |
++---+-------------+
+| sp|     x       |
```

Comment:
yes, that may work - but from what I've seen (empirically), `REQUIRED` constraints are faster to solve than anything else - so reducing the strength could screw up the performance. It's worth a shot.

And all right, will document it :)